test/integration/upload/cli/test_select.py

```diff
@@ -35,6 +35,17 @@ def test_when_given_an_unrecognized_urn_it_stores_it_in_upload_area_list_and_set
         self.assertEqual(self.uri, config.upload.areas[self.area_uuid]['uri'])
         self.assertEqual(self.area_uuid, config.upload.current_area)
 
+    def test_when_given_an_unrecognized_uri_without_slash_it_sets_it_as_current_area(self):
+        uri_without_slash = "s3://org-humancellatlas-upload-test/{}".format(self.area_uuid)
```

Comment:
Instance variables are generally preceded with an underscore [0]. If you're feeling gracious, could make the change throughout the class.

[0] https://www.python.org/dev/peps/pep-0008/#method-names-and-instance-variables